jobs/JGLOBAL_ARCHIVE

```diff
@@ -50,6 +50,10 @@ for grid in "0p25" "0p50" "1p00"; do
        "COMIN_ATMOS_GRIB_${grid}:COM_ATMOS_GRIB_GRID_TMPL"
 done
 
+for grid in "1p00" "5p00"; do
+    YMD=${PDY} HH=${cyc} GRID=${grid} declare_from_tmpl -rx \
+       "COMIN_OCEAN_NETCDF_${grid}:COM_OCEAN_NETCDF_GRID_TMPL"
+done
```

Comment:
This whole part should be conditional on `${DO_EXTRACTVARS}`, since that is the only job creating these files.

Reply:
Actually, none of this should be necessary now that the template filters are being used in the jinja file.